data/docs/instrumentation/angular.mdx

```diff
@@ -365,7 +365,7 @@ We all are familiar with instrumenting backend services but have you ever though
 
 #### Pre-requisites
 
-Enable CORS in the OTel Receiver. Inside `docker/clickhouse-setup/otel-collector-config.yaml` add the following CORS config. You can view the file at [SigNoz GitHub repo](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/otel-collector-config.yaml).
+Enable CORS in the OTel Receiver. Add the below required changes in the `otel-config.yml` file which can be generally found [here](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/otel-collector-config.yaml).
```

Comment:
```suggestion
Enable CORS in the OTel Receiver. Add the below required changes in the `otel-collector-config.yaml` file which can be found [here](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/otel-collector-config.yaml).
```

Reply:
`yaml` and `yml` are same.

Reply:
Yes. But in the link that we have mentioned, it uses `.yaml` file.
Let's maintain consistency.